docs/faq.md

```diff
@@ -224,6 +224,30 @@ Commandes utiles:
 
 Source du plugin: <https://github.com/NotSqrt/mattermost-integration-gitlab>
 
+### Mattermost Gitlab-CI integration
+
+Se rendre sur <https://mattermost.fabrique.social.gouv.fr/default/integrations/incoming_webhooks/add> et créer un webhook en sélectionant le canal sur lequel on souhaite publier les notifications. Récupérer l'url du webhook qui contiendra le token. Se rendre sur la page de configuration CI/CD du projet sur GitLab, example: https://gitlab.factory.social.gouv.fr/SocialGouv/emjpm/-/settings/ci_cd (remplacer emjpm par le nom du projet), et ajouter une variable intitulée `MATTERMOST_WEBHOOK` dans la sections `variables` et qui contiendra l'url du webhook.
+
+Dans le fichier `.gitlab-ci` à la racine du projet
+```yaml
+Ma notification mattermost:
+  image: registry.gitlab.factory.social.gouv.fr/devthejo/bash-curl:1.2.0
+  only:
+    refs:
+      - tags
+  script:
+    - |
+      echo '{"text":"🦊 release ready for production at '${CI_PIPELINE_URL}'"}' \
```

Comment:
```suggestion
      echo '{"text":"🦊 release ready for production at '${CI_ENVIRONMENT_URL}'"}' \
```

Reply:
du coup tu renvoi l'utilisateur sur le job qui lui fait la notif plutôt que sur le pipeline ?

Reply:
l'url du deploiement non ?

Reply:
je veux bien mais du coup il faut changer le texte pour "my job to do ... is running at" et rajouter `...` pour le reste du script

Reply:
🤔 "release ready for production at" je m'attendais à l'url du déploiement

Reply:
ha tu voulais probablement pointer vers le pipeline où il y a le bouton "trigger production"

Reply:
ca ira bien comme exemple btw

Reply:
exactement c'est vrai que c'était clair pour moi mais je suis dans mon context, c'est comme tu veux